src/search/landmarks/landmark_factory_relaxation.h

```diff
@@ -0,0 +1,47 @@
+#ifndef LANDMARKS_LANDMARK_FACTORY_RELAXATION_H
+#define LANDMARKS_LANDMARK_FACTORY_RELAXATION_H
+
+#include "landmark_factory.h"
+
+namespace landmarks {
+class Exploration;
+
+class LandmarkFactoryRelaxation : public LandmarkFactory {
+protected:
+    explicit LandmarkFactoryRelaxation(const options::Options &opts)
+        : LandmarkFactory(opts) {}
+
+    bool relaxed_task_solvable(const TaskProxy &task_proxy, Exploration &exploration,
+                               bool level_out,
+                               const LandmarkNode *exclude,
+                               bool compute_lvl_op = false) const;
+    bool relaxed_task_solvable(const TaskProxy &task_proxy, Exploration &exploration,
+                               std::vector<std::vector<int>> &lvl_var,
+                               std::vector<utils::HashMap<FactPair, int>> &lvl_op,
+                               bool level_out,
+                               const LandmarkNode *exclude,
+                               bool compute_lvl_op = false) const;
+
+private:
+    void generate_landmarks(const std::shared_ptr<AbstractTask> &task) override;
+
+    virtual void generate_landmarks(const std::shared_ptr<AbstractTask> &task,
+                                    Exploration &exploration) = 0;
```

Comment:
I was surprised to see that using the same name works here, but the signature is different enough, apparently. I'm not sure if having the same name is a good way to go, though. But I'm also not against it.

Reply:
I renamed this to ```generate_relaxed_landmarks``` to avoid confusion (if not to the compiler, then to us)